build_daemon/lib/src/managers/build_target_manager.dart

```diff
@@ -22,22 +22,23 @@ class BuildTargetManager {
 
   bool Function(BuildTarget, Iterable<WatchEvent>) shouldBuild;
 
-  bool get isEmpty => _buildTargets.isEmpty;
-
-  Set<BuildTarget> get targets => _buildTargets.keys.toSet();
-
-  Set<WebSocketChannel> channels(BuildTarget target) => _buildTargets[target];
```

Comment:
Just did an auto sort here.

Reply:
Try to keep these in separate commits if you can, it makes reviewing a lot easier :D